### PR TITLE
[FR] Add `Skinning/FAQ`, Update `Skinning/Interface`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page `FAQ`.

Proposal to update the French translation of the wiki page `Interface`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/Skinning/Interface/fr.md [hunk 1]
```diff
@@ -1030,680 +1076,681 @@ Notes:
...
+- Le positionnement varie :
+  - 192px à partir du bord droit de l'écran
+  - v1.0 : à 272px de hauteur
+  - v2.0+ : à 320px de hauteur
 
 ---
 
 `ranking-XH-small.png`
 
+
```
Comment (DoryanR) [inline wiki/Skinning/Interface/fr.md:1487]:
```suggestion
```

File: wiki/Skinning/FAQ/fr.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
+# FAQ sur le skinning
+
+Cet article traite des questions fréquemment posées sur le skinning.
+
+## Pour commencer
+
+### Qu'est-ce que le skinning ?
+
+Le skinning est un simple mécanisme de changement d'image qu'osu! chargera lors de la sélection d'un skin dans les options. Il est facile de créer un skin, mais il est un peu plus difficile de créer les ressources nécessaires.
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:9]:
```suggestion
Le skinning est un simple mécanisme de changement d'images qu'osu! chargera lors de la sélection d'un skin dans les options. Il est facile de créer un skin, mais il est un peu plus difficile de créer les ressources nécessaires.
```

File: wiki/Skinning/FAQ/fr.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+
+### Quelqu'un peut-il faire ce skin à partir de cette série/ce jeu ?
+
+Si vous avez parcouru l'ensemble des forums de skinning et que vous êtes certain de ne pas trouver *ce* skin, alors félicitations, osu! ne l'a pas encore. En gardant cela à l'esprit, vous pouvez prendre cette initiative pour créer le skin dont vous rêvez et en faire quelque chose que tout le monde peut vouloir ! 
+
+**Néanmoins, ne demandez jamais de skin sur les forums**. Le non-respect de cette règle entraînera le déplacement de votre fil de discussion vers wastelands.
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:15]:
```suggestion
**Néanmoins, ne demandez jamais de skin sur les forums**. Le non-respect de cette règle entraînera le déplacement de votre fil de discussion vers les wastelands.
```

File: wiki/Skinning/FAQ/fr.md [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+
+### Où puis-je obtenir ce skin avec lequel j'ai vu quelqu'un jouer ?
+
+Tout d'abord, ne posez pas cette question dans les forums ; sinon, votre fil de discussion sera très probablement jeté dans les wastelands. Ensuite, il est préférable de demander à cette personne le skin qu'elle utilise. Elle peut vous le donner, mais ce n'est pas toujours le cas.
+
+### Puis-je utiliser un élément skinné à partir du skin de quelqu'un d'autre ?
+
+Oui, vous pouvez utiliser des éléments de skinning provenant d'autres skins. Ces éléments sont appelés "skins mixtes" ou "skin mashups". Cependant, **vous n'êtes pas autorisé à les poster sur le sous-forum skinning, avec ou sans permission**, mais vous pouvez les poster dans votre signature ou sur votre page d'utilisateur moi!
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:23]:
```suggestion
Oui, vous pouvez utiliser des éléments de skinning provenant d'autres skins. Ces éléments sont appelés "skins mixtes" ou "skin mashups". Cependant, **vous n'êtes pas autorisé à les poster sur le sous-forum skinning, avec ou sans permission**, mais vous pouvez les poster dans votre signature ou sur votre page d'utilisateur "moi!"
```

File: wiki/Skinning/FAQ/fr.md [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+
+Cependant, lors de la publication de votre skin, vous devez supprimer ces fichiers afin de réduire la taille du fichier. Si vous utilisez le skin modèle, supprimez les éléments qui ne sont pas modifiés car osu! se chargera automatiquement des éléments manquants.
+
+Notez cependant que le dossier appelé `taiko` (insensible à la casse) est réservé aux skins osu!taiko qui remplaceront tous les éléments osu!taiko (et certains éléments d'interface lors de la lecture d'une beatmap osu!taiko).
+
+### Comment recharger mon skin dans osu! ?
+
+Vous devrez avoir votre skin sélectionné, puis appuyer sur ces touches pour recharger un skin dans osu! : `Ctrl` + `Alt` + `Shift` + `S`.
+
+Vous pouvez également quitter et rouvrir osu! ou lancer l'updater pour recharger le skin, mais il est généralement plus rapide d'utiliser la combinaison de quatre clés pour recharger le skin dans osu!
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:39]:
```suggestion
Vous pouvez également quitter et rouvrir osu! ou lancer l'updater pour recharger le skin, mais il est généralement plus rapide d'utiliser la combinaison de quatre touches pour recharger le skin dans osu!
```

File: wiki/Skinning/FAQ/fr.md [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+
+### Que sont les skinning sets ?
+
+*Page principale : [Ranking Criteria/Skin Set List](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+Dans osu!, il y a plus de 200 éléments qui peuvent être skinnés (sans compter les images animées individuelles). Cependant, vous n'êtes pas obligé de les skinner tous. Vous pouvez simplement les supprimer car osu! utilisera les images par défaut pour ceux que vous n'avez pas inclus.
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:45]:
```suggestion
Dans osu!, il y a plus de 200 éléments qui peuvent être skinnés (sans compter les images animées individuelles). Cependant, vous n'êtes pas obligé de tous les skinner. Vous pouvez simplement les supprimer car osu! utilisera les images par défaut pour ceux que vous n'avez pas inclus.
```

File: wiki/Skinning/FAQ/fr.md [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+
+Ceci étant dit, un ensemble de skinning (défini par les critères de classement) indique fondamentalement quels éléments doivent être skinnés si l'un d'entre eux dans l'ensemble l'était. Par exemple, le joueur qui utilise votre skin peut ne pas aimer si vous n'avez skinné que la moitié des numéros par défaut et laissé les autres utiliser leurs valeurs par défaut. Cependant, ceci ne s'applique strictement qu'au skinning des beatmaps mais vous pouvez en tenir compte pour votre propre skin.
+
+### Une skin éditée est-elle un skin mixte ?
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:49]:
```suggestion
### Un skin édité est-il un skin mixte ?
```

File: wiki/Skinning/FAQ/fr.md [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+
+Les skin édités sont considérés comme mixtes. Ils ne peuvent donc être postés dans aucun sous-forum.
+
+### Puis-je refaire un skin que quelqu'un d'autre a fait ?
+
+Oui, vous pouvez avoir un thème de skin similaire à celui de quelqu'un d'autre qui l'a peut-être fait avant vous. Cependant, vous devez créer les actifs vous-même et, évidemment, ne pas copier les actifs de l'autre skin que vous essayez de refaire.
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:55]:
```suggestion
Oui, vous pouvez avoir un thème de skin similaire à celui de quelqu'un d'autre qui l'a peut-être fait avant vous. Cependant, vous devez créer les éléments vous-même et, évidemment, ne pas copier les éléments de l'autre skin que vous essayez de refaire.
```

File: wiki/Skinning/FAQ/fr.md [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+
+### Pourquoi n'y a-t-il pas une base de données pour les skins, comme les beatmaps ?
+
+Il existe des projets tiers qui tentent déjà de le faire. Cependant, la base de données de skinning est déjà prévue. Pour plus de détails, voir [ppy/osu-web#61 - Skin System](https://github.com/ppy/osu-web/issues/61).
+
+## Partage
+
+### Comment puis-je partager mon skin ?
+
+Tout d'abord, avant de partager votre skin, assurez-vous que vous avez supprimé les autres fichiers qui ne sont pas utilisés. Il peut s'agir d'images et de sons par défaut, de dossiers supplémentaires, de fichiers non-image (à l'exception du fichier `skin.ini`). Une fois cette étape terminée, vous devrez exporter votre skin sous forme de fichier `.osk`. Avec ce fichier, vous pouvez le télécharger et le poster dans le sous-forum skinning.
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:65]:
```suggestion
Tout d'abord, avant de partager votre skin, assurez-vous que vous avez supprimé les autres fichiers qui ne sont pas utilisés. Il peut s'agir d'images et de sons par défaut, de dossiers supplémentaires, de fichiers non-image (à l'exception du fichier `skin.ini`). Une fois cette étape terminée, vous devrez exporter votre skin sous forme de fichier `.osk`. Avec ce fichier, vous pouvez le mettre en ligne et le poster dans le sous-forum skinning.
```

File: wiki/Skinning/FAQ/fr.md [hunk 10]
```diff
@@ -0,0 +1,294 @@
...
+
+1. Ouvrez osu!.
+2. Ouvrez la barre latérale des options.
+3. Chargez votre skin.
+4. Cliquez sur `Exporter en .osk`.
+5. A partir de là, osu! ouvrira une fenêtre pour votre skin exporté.
+6. Prenez le fichier `.osk` avec le nom de votre skin et téléchargez-le.
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:72]:
```suggestion
6. Prenez le fichier `.osk` avec le nom de votre skin et mettez-le en ligne.
```

File: wiki/Skinning/FAQ/fr.md [hunk 11]
```diff
@@ -0,0 +1,294 @@
...
+7. Vous pouvez maintenant le partager !
+
+### Puis-je poster un skin mixte dans les sous-forums de skinning ?
+
+Non, vous n'êtes pas autorisé à poster un skin mixte dans aucun sous-forum. La même chose s'applique aux skin édités.
+
+#### Où puis-je afficher mon skin mixte ?
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:79]:
```suggestion
#### Où puis-je poster mon skin mixte ?
```

File: wiki/Skinning/FAQ/fr.md [hunk 12]
```diff
@@ -0,0 +1,294 @@
...
+
+Vous pouvez utiliser votre page d'utilisateur moi!, votre signature, le [/r/osuskins subreddit](https://www.reddit.com/r/OsuSkins), ou tout autre endroit en dehors des forums d'osu!
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:81]:
```suggestion
Vous pouvez utiliser votre page d'utilisateur "moi!", votre signature, le [subreddit /r/osuskins](https://www.reddit.com/r/OsuSkins), ou tout autre endroit en dehors des forums d'osu!
```

File: wiki/Skinning/FAQ/fr.md [hunk 13]
```diff
@@ -0,0 +1,294 @@
...
+
+### Puis-je poster un remake d'un skin ?
+
+Oui, les remakes de skin ne sont pas considérés comme mixtes. Vous pouvez donc les poster, tant qu'aucun des actifs n'a été copié ou dupliqué à partir d'un autre skin (auquel cas, il serait considéré comme mixte).
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:85]:
```suggestion
Oui, les remakes de skin ne sont pas considérés comme mixtes. Vous pouvez donc les poster, tant qu'aucun des éléments n'a été copié ou dupliqué à partir d'un autre skin (auquel cas, il serait considéré comme mixte).
```

File: wiki/Skinning/FAQ/fr.md [hunk 14]
```diff
@@ -0,0 +1,294 @@
...
+
+### Puis-je poster des skinnés R15 dans les sous-forums de skinning ?
+
+**Oui, et si vous le faites, vous devez ajouter un avertissement sur votre post de forum.** Les skinings R15 peuvent inclure, mais ne sont pas limités à :
+
+- des sous-vêtements suggestifs ou visibles
+- des maillots de bain et/ou bikinis
+- des personnages portant des vêtements révélateurs ou peu voyants
+
+Pour ajouter cet avertissement, il suffit de préfixer `[R15]` au titre de votre fil de discussion et d'écrire un avis en haut du post lui-même.
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:95]:
```suggestion
Pour ajouter cet avertissement, il suffit de préfixer `[R15]` au titre de votre post et d'écrire un avis en haut du post lui-même.
```

File: wiki/Skinning/FAQ/fr.md [hunk 15]
```diff
@@ -0,0 +1,294 @@
...
+
+## Images
+
+### Quel doit être le format des images ?
+
+Dans la plupart des cas, les images doivent utiliser le format `.png`. Toutefois, l'élément `menu-background.jpg` doit utiliser l'extension `.jpg`.
+
+En plus d'utiliser le format `.png`, vous devriez :
+
+- Découpez ou recadrez vos images autant que possible. Le client d'osu! rendra chaque pixel d'une image, ce qui entraînera une charge de travail plus importante !
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:164]:
```suggestion
- Découper ou recadrer vos images autant que possible. Le client d'osu! rendra chaque pixel d'une image, ce qui entraînera une charge de travail plus importante !
```

File: wiki/Skinning/FAQ/fr.md [hunk 16]
```diff
@@ -0,0 +1,294 @@
...
+  - Notez cependant qu'une certaine transparence dans certaines sections d'une image est nécessaire pour aligner correctement des éléments spécifiques.
+- Compressez les images chaque fois que possible. La compression supprime généralement les informations inutiles sur les pixels vides et réduit considérablement la taille des fichiers.
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:166]:
```suggestion
- Compresser les images chaque fois que possible. La compression supprime généralement les informations inutiles sur les pixels vides et réduit considérablement la taille des fichiers.
```

File: wiki/Skinning/FAQ/fr.md [hunk 17]
```diff
@@ -0,0 +1,294 @@
...
+
+*Remarque : tous les éléments n'ont pas un trait d'union (`-`) entre le nom et le numéro de l'image. Consultez la liste des éléments pour connaître les noms des animations.*
+
+Les fréquences d'images de l'animation pour chaque élément sont incohérentes. Certains utilisent une valeur définie dans le fichier skin.ini, d'autres utilisent la valeur définie par le client, tandis que d'autres encore sont basés sur le BPM (battements par minute) de la beatmap. Pour cette raison, l'animation des éléments de skin se fait par essais et erreurs (voir la section suivante pour les méthodes de résolution de ce problème).
+
+### Mon animation est trop rapide/lente, comment y remédier ?
+
+Si votre animation est trop rapide/lente, vous avez deux façons de la corriger :
+
+1. Si elle joue trop vite, essayez de les allonger en doublant les images (ou en les divisant par deux, si elle est trop lente).
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:231]:
```suggestion
1. Si elle se joue trop vite, essayez de les allonger en doublant les images (ou en les divisant par deux, si elle est trop lente).
```

File: wiki/Skinning/FAQ/fr.md [hunk 18]
```diff
@@ -0,0 +1,294 @@
...
+
+### J'ai supprimé un élément, mais il est toujours là ! Comment puis-je le supprimer ??
+
+Si vous voulez supprimer un élément, vous devez utiliser une image transparente à la place de l'élément que vous voulez supprimer. Ensuite, [rechargez votre skin](#comment-recharger-mon-skin-dans-osu!-?).
+
+#### Comment supprimer l'éclairage après avoir tapé sur un objet ?
+
+Il y a deux façons de procéder :
+
+- Désactiver l'éclairage des frappes dans les [options](/wiki/Options).
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:251]:
```suggestion
- Désactiver les lueurs de fond dans les [options](/wiki/Options).
```

File: wiki/Skinning/FAQ/fr.md [hunk 19]
```diff
@@ -0,0 +1,294 @@
...
+
+Pour les particules, supprimez simplement les fichiers `particle50.png`, `particle100.png`, et `particle300.png`. Vous n'avez pas besoin d'une image transparente pour ces trois fichiers.
+
+#### Comment retirer le slider end du cercle ?
+
+Ajoutez un fichier transparent appelé `sliderendcircle.png` à votre skin.
+
+#### Comment puis-je cacher les scores de 300 points ?
+
+Ajouter un fichier transparent pour ces noms de fichiers :
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:264]:
```suggestion
Ajoutez un fichier transparent pour ces noms de fichiers :
```

File: wiki/Skinning/Interface/fr.md [hunk 20]
```diff
@@ -1,191 +1,202 @@
...
-- Cet élément est utilisé comme playfield si la beatmap ne possède pas de fond d'écran.
-- L'option du fond d'écran saisonnier peut affecter la visibilité de cet élément.
-  - Si elle est réglée sur "Toujours", les fonds d'écran saisonniers vont remplacer cet élément.
-  - Si elle est réglée sur "Parfois", les fonds d'écran saisonniers vont remplacer cet élément tant qu'ils sont actifs.
-- Extension `.jpg` uniquement.
-  - Si l'image est de type `.png`, changer l'extension en `.jpg`.
-    - Si le fond d'écran est transparent, la couleur de fond sera noire. 
-- Le joueur peut glisser-déposer une image pour écraser l'image du skin. **Ceci va remplacer l'image dans le répertoire du skin !**
+- Tag [osu!supporter](/wiki/osu!supporter) requis.
+- Cet élément est positionné au centre et est configuré pour couvrir (remplir toute la largeur et la hauteur tout en gardant son rapport d'aspect, mais rogner tout ce qui est en dehors de la fenêtre du jeu).
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:20]:
```suggestion
- Cet élément est positionné au centre et est configuré pour couvrir (remplir toute la largeur et la hauteur tout en gardant son aspect ratio, mais rogner tout ce qui est en dehors de la fenêtre du jeu).
```

File: wiki/Skinning/Interface/fr.md [hunk 21]
```diff
@@ -1,191 +1,202 @@
...
+- Par défaut, osu! dispose d'un ensemble d'images d'arrière-plan qu'il fait défiler.
+  - Si cet élément est skinné et que l'utilisateur a le tag osu!supporter, cet élément annulera ce comportement.
+- Cet élément est utilisé comme terrain de jeu si la beatmap n'a pas de fond.
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:23]:
```suggestion
- Cet élément est utilisé comme playfield si la beatmap n'a pas de fond.
```

File: wiki/Skinning/FAQ/fr.md [hunk 22]
```diff
@@ -0,0 +1,294 @@
...
+
+Il est préférable de garder à l'esprit les résolutions suivantes lors du skinning :
+
+- 1024x768 (4:3, résolution standard du jeu, le jeu est basé sur celle-ci)
+- 2048x1536 (4:3, résolution standard du jeu en mise à l'échelle HD)
+- 1366x768 (16:9, résolution standard du jeu en écran large)
+- 2732x1536 (16:9, résolution écran large du jeu standard en mise à l'échelle HD)
+- 1920x1080 (16:9, résolution HD standard)
+
+Les images seront ajustées par le jeu lui-même pour s'adapter à des résolutions dérivées de celles mentionnées ci-dessus. La plupart d'entre elles seront redimensionnées pour s'adapter au champ de jeu ou repositionnées sur des rapports d'aspect différents.
```
Comment (Ganondorf-1) [inline wiki/Skinning/FAQ/fr.md:184]:
```suggestion
Les images seront ajustées par le jeu lui-même pour s'adapter à des résolutions dérivées de celles mentionnées ci-dessus. La plupart d'entre elles seront redimensionnées pour s'adapter au playfield ou repositionnées sur des rapports d'aspect différents.
```

File: wiki/Skinning/Interface/fr.md [hunk 23]
```diff
@@ -554,107 +574,133 @@ Notes:
...
+| Toutes | ![Non][false] | ![Non][false] | Multiplicatif | Centre | - |
 
-Notes:
+Notes :
 
-- La teinte varie en fonction de l'état du tick.
+- Les teintes varient en fonction de l'état du tick.
 
-## Playfield
+## Terrain de jeu
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:628]:
```suggestion
## Playfield
```

File: wiki/Skinning/Interface/fr.md [hunk 24]
```diff
@@ -944,84 +990,84 @@ Notes:
...
-Notes:
+Notes :
 
-- Quand le jeu sera mis en pause, la zone de jeu sera assombrie et ce fichier s'y superposera. 
-- Cet élément ne sera pas déformé.
-- La hauteur d'image entière est de 768px.
-- Les images plus petites sont affichées avec des bordures transparentes, tandis que les images plus grandes sont recoupées.
-- Peut également être au format `.jpg` (et avoir l'extension `.jpg`).
-  - osu! privilégie le format `.png` au format `.jpg`.
+- Lorsque le jeu est en pause, le champ de jeu sera obscurci et ce fichier se superposera en haut de celui-ci.
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:1023]:
```suggestion
- Lorsque le jeu est en pause, le playfield sera obscurci et ce fichier se superposera en haut de celui-ci.
```

File: wiki/Skinning/Interface/fr.md [hunk 25]
```diff
@@ -2315,197 +2361,197 @@ Notes:
...
+| Toutes | ![Non][false] | ![Non][false] | Additif | Centre | 32x32 |
 
-Notes:
+Notes :
 
-- Cet élément est placé au dessus de `selection-mode.png`.
-- Sélectionner [osu!mania](/wiki/Game_mode/osu!mania) pour voir l'élément.
-- Si `menu-snow.png` n'est pas présent, cet élément sera utilisé si il est sélectionné.
+- Cet élément se trouve en haut de l'élément `selection-mode.png`.
+- Select [osu!mania](/wiki/Game_mode/osu!mania) for this to be visible.
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:2553]:
```suggestion
- Sélectionnez [osu!mania](/wiki/Game_mode/osu!mania) pour que cela soit visible.
```

File: wiki/Skinning/Interface/fr.md [hunk 26]
```diff
@@ -1030,680 +1076,680 @@ Notes:
...
 
-Notes:
+Notes :
 
-- Origine variable :
-  - Break : Centre
-  - Écran de sélection des beatmaps : Gauche
-  - Classement : Centre
+- L'origine varie :
+  - Pause :  Centre
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:1530]:
```suggestion
  - Pause : Centre
```

File: wiki/Skinning/Interface/fr.md [hunk 27]
```diff
@@ -1030,680 +1076,680 @@ Notes:
...
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:1564]:
```suggestion
  - Pause : Centre
```

File: wiki/Skinning/Interface/fr.md [hunk 28]
```diff
@@ -1030,680 +1076,680 @@ Notes:
...
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:1598]:
```suggestion
  - Pause : Centre
```

File: wiki/Skinning/Interface/fr.md [hunk 29]
```diff
@@ -1030,680 +1076,680 @@ Notes:
...
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:1632]:
```suggestion
  - Pause : Centre
```

File: wiki/Skinning/Interface/fr.md [hunk 30]
```diff
@@ -1030,680 +1076,680 @@ Notes:
...
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:1666]:
```suggestion
  - Pause : Centre
```

File: wiki/Skinning/Interface/fr.md [hunk 31]
```diff
@@ -1030,680 +1076,680 @@ Notes:
...
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:1700]:
```suggestion
  - Pause : Centre
```

File: wiki/Skinning/Interface/fr.md [hunk 32]
```diff
@@ -1030,680 +1076,680 @@ Notes:
...
```
Comment (Ganondorf-1) [inline wiki/Skinning/Interface/fr.md:1734]:
```suggestion
  - Pause : Centre
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@Ganondorf-1 recheck please

**Review by TicClick — APPROVED** [review]
merging with the assumption that one review was enough

**Pull request merged**